CMakeLists.txt

```diff
@@ -501,13 +501,31 @@ endif()
 # BUILTINS
 #----------------------------------------------------------------------------#
 
-include(CudaRdcUtils)
-
 if(CELERITAS_BUILTIN)
   include(FetchContent)
   add_subdirectory(external)
 endif()
 
+#----------------------------------------------------------------------------#
+# RDC SUPPORT FOR VECGEOM+CUDA
+#----------------------------------------------------------------------------#
+
+# Load RDC utils using CMAKE_MODULE_PATH
+include(CudaRdcUtils OPTIONAL)
+
+# Load our local copy if it's newer
+if(NOT CUDA_RDC_VERSION)
+   set(_loaded_CUDA_RDC_VERSION 0)
+else()
+   set(_loaded_CUDA_RDC_VERSION ${CUDA_RDC_VERSION})
```

Comment:
Wait since now we remembered that `LibraryUtils` loads our own copy of CudaRdcUtils first, won't this always be set and "up to date" (even if a previous older version was later loaded and overrode our functions with old versions?) ?

Reply:
Humm .. no?

> Wait since now we remembered that LibraryUtils loads our own copy of CudaRdcUtils first,

It does not (in the current state).  It loads one on the CMAKE_MODULE_PATH.